app/services/transaction_service/transaction.rb

```diff
@@ -48,22 +48,14 @@ def can_start_transaction_braintree(community_id:, author_id:)
   end
 
   def can_start_transaction_paypal(community_id:, author_id:)
-    person_account_response = paypal_account_api.get(
-      community_id: community_id,
-      person_id: author_id
-    )
-    personal_account_active = Maybe(person_account_response)[:data][:active]
-
-    admin_account_response = paypal_account_api.get(
-      community_id: community_id
-    )
-    admin_account_active = Maybe(admin_account_response)[:data][:active]
+    personal_account_prepared = PaypalHelper.account_prepared_for_user?(author_id, community_id)
```

Comment:
That's a good suggesting. For the sake of consistency, let's make it `state: one_of [:verified, :not_verified]`

Then another point: `active` is actually only used by the PayPal API to internally select which account to return, so I guess there's no reason why we would want to expose that. Anyway, it's always `true`.